### ISSUE TITLE
Crash when trying to copy paste text from the Run > Executable text field into the working directory text field: ig_text_callback_resize

### ISSUE THREAD

**Issue by DerTee** [WhiteBox-Systems/whitebox#32]
### What happened?

Whitebox crashes when trying to copy paste text from the Run > Executable text field into the working directory text field.

Assertion text:
```
Internal Error: buffers should match

Assertion failed in `ig_text_callback_resize` - gui_primitives.c(975):
    buf_ptr[0]->arr == data->Buf
    last OS error => 0: The operation completed successfully.
Rich assert failed (hr: 0x80004005, comctl: 00007FFB72890000, TDI: 0000000000000000), using fallback...
```

### System/app details

sys_info:
WhiteBox version - 2024-10-22
WhiteBox commit  - 137532b2476839ab2ecc52324866745ec77ed0a8
Assert level     - 1
Debug info       - false
OS               - Windows 10 Home x64 (10.0.22631)
Service pack     - (0.0)
CPU              - AMD Ryzen 5 2600 Six-Core Processor            
Logical cores    - 12
CPU revision     - 0x802
RAM              - 16.0 GiB
Address space    - 128.0 TiB

### How to reproduce the error

Steps to reproduce:
 1. Open whitebox_nightly-2024-10-22
 2. click on Run > Executable and select the executable
 3. copy the file path next to the executable button (because I want the working directory to be the dir of the exe 99% of the time)
 4. paste into the working directory text field -> CRASH assertion failure

Does the error happen consistently with the above steps?
Always

### Crash/log attachments & additional info

- [X] crash dump file (much smaller if (g)-zipped)
- [ ] whitebox-log.txt file

[whitebox-2024-10-28-19-45-33.dmp.gz](https://github.com/user-attachments/files/17548018/whitebox-2024-10-28-19-45-33.dmp.gz)

**Comment by azmr** [comment]
Thanks for these reports, I'll have a look tomorrow morning and see if I can repro.

As you say re working directory, these are in dire need of a UX pass anyway!

**Comment by azmr** [comment]
Fixed in `8aafa787`
